client/components/date-range/index.js

```diff
@@ -400,20 +399,14 @@ export class DateRange extends Component {
 		return window.matchMedia( '(min-width: 520px)' ).matches ? 2 : 1;
 	}
 
-	handleDateRangeChange = ( startDate, endDate, shortcutId = '' ) => {
+	handleDateRangeChange = ( startDate, endDate, dateClicked ) => {
 		this.setState( {
 			startDate,
 			endDate,
 			textInputStartDate: this.toDateString( startDate ),
 			textInputEndDate: this.toDateString( endDate ),
-			currentShortcut: shortcutId,
 		} );
-		this.props.onDateSelect && this.props.onDateSelect( startDate, endDate );
-	};
-
-	handleCalendarChange = ( startDate, endDate ) => {
-		// When the calendar or inputs change directly, set to custom range
-		this.handleDateRangeChange( startDate, endDate, 'custom_date_range' );
+		this.props.onDateSelect && this.props.onDateSelect( dateClicked );
```

Comment:
`onDateSelect` should accept a single date as param by its naming, and we don't want to change that.